catalog/video/video-mpegts-transformer/src/main/resources/DDF_Custom_Mime_Type_Resolver-mpegts.config

```diff
@@ -1,7 +1,7 @@
-customMimeTypes=[ \
-  "ts\=video/mp2t",
-  "mpg\=video/mp2t",
-  "mpeg\=video/mp2t"
+customMimeTypes = [ \
+  "ts\=video/mp2t", \
+  "mpg\=video/mp2t", \
+  "mpeg\=video/mp2t", \
```

Comment:
✏️  Remove trailing `,` to avoid confusion.

Reply:
I decided to leave the extra comma and include a note in the Documentation. See https://github.com/codice/ddf/pull/2241/commits/5e3264137917b6ce6ffed300a59c64405933b8e4#r135342949.